src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -650,23 +661,23 @@ void func_80B0DB00(EnSw* this, PlayState* play) {
 
         this->actor.bgCheckFlags &= ~BGCHECKFLAG_GROUND;
 
-        if (this->unk_38A == 0) {
-            this->actionFunc = func_80B0DC7C;
-            this->unk_394 = 10;
+        if (this->animationSpeed == 0) {
+            this->actionFunc = EnSw_DieNormal;
+            this->deathFlames = 10;
         } else {
-            this->actor.velocity.y = ((this->unk_38A--) * 8.0f) * 0.5f;
+            this->actor.velocity.y = ((this->animationSpeed--) * 8.0f) * 0.5f;
```

Comment:
```suggestion
            this->actor.velocity.y = (this->animationSpeed * 8.0f) * 0.5f;
            this->animationSpeed--;
```
This still works and would be my preference.